docs/api-docs/partner/subscriptions/foundation-guide.md

```diff
@@ -0,0 +1,247 @@
+
+We've designed this integration to use Stripe Connect so that your app can use one connection to submit payments to multiple merchants' previously configured Stripe accounts. See [Stripe Billing](https://support.bigcommerce.com/s/article/Connecting-Stripe-Payment-Gateway?language=en_US#billing) for merchant configuration instructions.
+
+The respective architectures of the Stripe and BigCommerce APIs on which this app relies require the app to make charges using the **merchant-specific Stripe account's public key** and the **app-specific Stripe account's secret key**. 
+
+This arrangement provides the following benefits:
+* The merchant can change the Stripe account they use without breaking the integration.
+* The merchant's own Stripe account stores all shopper payment history, subscription and one-time purchase alike. 
+* It limits the app's PCI compliance burden. 
+
+In development, you'll work with two Stripe accounts; one that simulates a merchant's pre-existing account, and another that you configure as the app's dedicated Stripe Connect-enabled account.
+
+<!-- theme: info -->
+> #### Multi-tenant mechanics
+> During installation, the app UI prompts the merchant to enter their Stripe account's keys so that the app can initiate an OAuth grant code authorization flow that links the merchant's Stripe account to the app's Stripe Connect-enabled account. 
+
+### Create Stripe accounts
+
+To get started, do the following steps:
+
+1. To create the Stripe account that simulates your merchant's pre-existing Stripe account, [sign up with Stripe](https://dashboard.stripe.com/register). Your app uses this account's client ID to process subscriber payments and manage subscriptions; give it a merchant-specific name.
+
+2. Sign in to the [Stripe Dashboard](https://dashboard.stripe.com/).
+
+3. From inside the [Stripe Dashboard](https://dashboard.stripe.com/), create and configure a second app-specific Stripe account. The app uses this account's client secret to process subscription payments.
+
+   a. On the left side of your Stripe Dashboard's top menu bar, click the name of your merchant-specific account. In the dropdown menu that appears, select **+ New Account**.
+
+   b. Give the new account a name that clearly marks it as app-specific; the idea is to differentiate it from the merchant-specific payment and subscription management account. This guide uses **BigCommerce App**.
+   
+   ![BigCommerce App](https://storage.googleapis.com/bigcommerce-production-dev-center/images/BigCommerce-app-image.png "The Stripe dashboard's new account dropdown menu")
+   
+   c. Under **Get started with Stripe**, copy the **Secret key**. In a later step, you will use the secret key to update the `STRIPE_SECRET_KEY` environment variable in the .env file.
+
+### Enable Stripe Connect for Platforms
+
+**During this instruction sequence, make sure that the left side of your Stripe Dashboard's top menu bar indicates that you're in the app-specific account.**
+
```

Comment:
This sentence sounds a little weird. I would say, before you start the steps below, make sure that you are logged into the app-specific Stripe account. Your account information is displayed on the top left corner of the menu bar of the Stripe Dashboard. (Maybe add screenshot)